docs/src/js/index.js

```diff
@@ -3,5 +3,6 @@ import '../../../components/menu/src/index.js';
 import '../../../components/content-navigation/src/index.js';
 import '../../../components/plus/index.js';
 import '../../../components/minus/index.js';
-import '../../../components/accordion/dist/index.js';
-import './component-sidebar.js';
+// import '../../../components/accordion/dist/index.js'; // Causes debugger to appear
+// import './component-sidebar.js'; // Produces JS breaking null error
```

Comment:
Are these lines still causing problems? I put in a fix for at least the `component-sidebar.js` problem a few days ago.